plugins/hdfs_assetstore/plugin_tests/assetstore_test.py

```diff
@@ -54,8 +55,8 @@ def serverdefaults(self):
 
     def df(self):
         return {
-            'used': 100L,
-            'capacity': 1000L
+            'used': 100,
```

Comment:
@zachmullen This was causing the style checks to fail in Python 3. Since there is no ``long`` type or literals in Python 3, and since ``six`` does not have any support for ``long``, I changed these to ``int`` literals. 

If there is an important reason to make them ``long`` in Python2, we can add some extra logic.